io/read.go

```diff
@@ -56,7 +56,25 @@ func consistentReadSync(filename string, attempts int, sync func(int)) ([]byte,
 		// Files are different, continue reading
 		oldContent = newContent
 	}
-	return nil, fmt.Errorf("could not get consistent content of %s after %d attempts", filename, attempts)
+	return nil, InconsistentReadError{filename, attempts}
```

Comment:
maybe add this new error type into function comment too?

Reply:
Added. And added a comment to `InconsistentReadError` type.